<?xml version="1.0"?>
<launch>
  <!-- static camera node -->
  <node pkg="atl_ros" name="atl_static_camera" type="atl_pg_camera_node" output="screen" required="true">
    <param name="guid" type="str" value="49712223533776454" />
    <param name="stamp_image" value="false" />
    <param name="image_topic" value="/atl/static_camera/image" />
    <param name="config_dir" value="$(find atl_configs)/configs/camera/pointgrey_firefly2" />
  </node>

  <!-- gimbal camera node -->
  <node pkg="atl_ros" name="atl_gimbal_camera" type="atl_pg_camera_node" output="screen" required="true" >
    <param name="guid" type="str" value="49712223533654162" />
    <param name="stamp_image" value="false" />
    <param name="image_topic" value="/atl/gimbal_camera/image" />
    <param name="config_dir" value="$(find atl_configs)/configs/camera/pointgrey_firefly" />
  </node>

  <!-- gimbal node -->
  <node pkg="atl_ros" name="atl_gimbal" type="atl_gimbal_node" output="screen" required="true" >
    <param name="gimbal_imu" value="SBGC" />
    <param name="config" value="$(find atl_configs)/configs/gimbal/config.yaml" />
  </node>

  <!-- cam calib node -->
  <node pkg="atl_ros" name="atl_cam_calib" type="atl_cam_calib_node" output="screen" required="true">
    <param name="calib_dir" value="/tmp/calibration" />
    <param name="chessboard_rows" value="8" />
    <param name="chessboard_cols" value="8" />
    <param name="static_camera_topic" value="/atl/static_camera/image" />
    <param name="gimbal_camera_topic" value="/atl/gimbal_camera/image" />
  </node>
</launch>
